kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/internal/VariantsImpl.kt

```diff
@@ -17,6 +17,7 @@ internal abstract class KoverCurrentProjectVariantsConfigImpl @Inject constructo
     KoverVariantConfigImpl(objects), KoverCurrentProjectVariantsConfig {
     internal val customVariants: MutableMap<String, KoverVariantCreateConfigImpl> = mutableMapOf()
     internal val providedVariants: MutableMap<String, KoverVariantConfigImpl> = mutableMapOf()
+    internal val copyVariants: MutableMap<String, String> = mutableMapOf()
```

Comment:
I guess it is `provided` because a variant already exists (provided by someone), but `copy` is an action we perform, just as `createVariant` — another way to add a new variant to the config.

Reply:
> I guess it is provided because a variant already exists (provided by someone), but copy is an action we perform, just as createVariant — another way to add a new variant to the config.

Something like this)

Reply:
The action is `copy` (and thus the function name), but the map contains the results of the action (well, or its inputs, then `variantsToCopy`) :)

Reply:
`copiedVariants` it is then.